src/components/GAddressLookup/component.js

```diff
@@ -0,0 +1,271 @@
+            this.autocompleteService.getPlacePredictions(
+                {
+                    ...searchOptions,
+                    input: needle,
+                },
+                (predictions, status) => {
+                    this.setState({ isSearching: false });
+
+                    if (status !== window.google.maps.places.PlacesServiceStatus.OK) {
+                        reject(status);
+                    }
+
+                    resolve(predictions);
+                },
+            );
+        });
+    }
+
+    processSearchResults(results) {
+        this.setState({
+            places: results,
+            suggestions: [
+                {
+                    label: (
+                        <span>
+                            Search for: <b>&lsquo;{this.state.searchValue}&rsquo;</b>
+                        </span>
+                    ),
+                    icon: <SearchValueIcon />,
+                },
+            ].concat(results.map(place => formatSuggestionItem(place, true, <LocationItemIcon />))),
+        });
+    }
+
+    handleError() {
+        this.setState({
+            places: [],
+            suggestions: [
+                {
+                    label: (
+                        <span>
+                            Search for: <b>&lsquo;{this.state.searchValue}&rsquo;</b>
+                        </span>
+                    ),
+                    icon: <SearchValueIcon />,
+                },
+            ],
+        });
+    }
+
+    clear() {
+        this.setState({
+            searchValue: '',
+            places: [],
+            suggestions: [],
+        });
+    }
+
+    render() {
+        const {
+            style,
+            label,
+            error,
+            placeholder,
+            disabled,
+            tabIndex,
+            required,
+            id,
+            name,
+            hideLabel,
+        } = this.props;
+        const { isSearching, suggestions } = this.state;
+        return (
+            <div id={id} style={style} className={this.getContainerClassName()}>
+                <Lookup
+                    id={this.lookupId}
+                    name={name}
+                    label={label}
+                    debounce
+                    isLoading={isSearching}
+                    placeholder={placeholder}
+                    options={suggestions.length > 0 ? suggestions : null}
+                    hideLabel={hideLabel}
+                    required={required}
+                    tabIndex={tabIndex}
+                    disabled={disabled}
+                    value={this.state.selectedPlace}
+                    onChange={this.handleChange}
+                    onSearch={this.handleSearch}
+                    error={error}
+                    icon={<LocationIcon />}
+                />
+                <div className="rainbow-google-address-poweredby_container">
+                    <img src={poweredByGoogleColored} alt="powered by Google" />
```

Comment:
The `Powered by Google` image should be aligned to the right. Please review the design. 
<img width="956" alt="Screen Shot 2019-06-06 at 12 44 09 AM" src="https://user-images.githubusercontent.com/32083556/59012897-53453c00-87fe-11e9-9f74-70cf33d42651.png">

It should be hidden when an error is showed.
<img width="758" alt="Screen Shot 2019-06-06 at 1 57 57 AM" src="https://user-images.githubusercontent.com/32083556/59012994-8e476f80-87fe-11e9-8de6-0859164a9081.png">